### ISSUE TITLE
VirusTotal - 'Telegram Scraper' - StemRoller (bin) in Release 2.0.3

### ISSUE THREAD

**Issue by ncolyer** [stemrollerapp/stemroller#57]
**TL;DR:**
_It's probably nothing but I figured it worthwhile to raise this based on increased activity around NPM packages (typo-naming packages, takeovers, etc)._

**Release:**
[Apple Release 2.0.3](https://github.com/stemrollerapp/stemroller/releases/tag/2.0.3#:~:text=2.0.3-,ef4978e,-Progress%20indicator%2C%20MP3)
StemRoller.App > Contents > MacOS
StemRoller - `ec13e1d9a6af478e4fdad876f08f53acf31bd136fac0be3c2698a460eda054a3`
_(Haven't checked any others)_

**Issue:**
The binary at `StemRoller.App > Contents > MacOS > StemRoller` (within the app package itself) is hash matching [Maestro Telegram Scraper](https://www.virustotal.com/gui/file/ec13e1d9a6af478e4fdad876f08f53acf31bd136fac0be3c2698a460eda054a3/detection).

It is not flagged as malicious likely due to legitimate utility in it's domain of use (scraping > communications) however in the context of an audio-splitting utility, is worthy further scrutiny unless the result of a niche runtime, hash collision, VirusTotal issue such as an index key:value / UX mismatch, or other reasonable explanation ( user error? : -) )

**Comment by ncolyer** [comment]
> Hmm... that binary should just be a repackaged copy of Electron, a commonly used framework for building desktop applications with HTML/CSS/JS. Do you know if Maestro Telegram Scraper also uses Electron?

I do not know the answer to that unfortunately, just the details VT provided.

**Comment by iffyloop** [comment]
No problem, I'll try to look into this sometime but it's most likely a false positive. Thanks for reporting.

**Comment by ncolyer** [comment]
Closing out, false positive. Result of shared electron.js runtime

**Comment by iffyloop** [comment]
Thanks for confirming @ncolyer
